xclim/indices.py

```diff
@@ -83,23 +93,27 @@ def base_flow_index(q, freq='YS'):
 @with_attrs(standard_name='cold_spell_duration_index', long_name='', units='days')
 @valid_daily_min_temperature
 def cold_spell_duration_index(tasmin, tn10, freq='YS'):
-    """Cold spell duration index.
+    r"""Cold spell duration index
 
-    Resample the daily minimum temperature series by returning the number of days per
+    Re-samples the daily minimum temperature series by returning the number of days per
     period where the temperature is below the calendar day 10th percentile (calculated
     over a centered 5-day window for values during the 1961–1990 period) for a minimum of
     at least six consecutive days.
 
     Parameters
     ----------
     tasmin : xarray.DataArray
-      Minimum daily temperature values [K].
+      Minimum daily temperature values [C] or [K]
     tn10 : xarray.DataArray
       The climatological 10th percentile of the minimum daily temperature computed over
-      the period 1961-1990, computed for each day of the year using a centered 5-day window.
```

Comment:
I suggest we use grammatically correct sentences whenever possible.

Reply:
Ie with punctuation. I suggest that if clause has a verb, use punctuation, otherwise not.